.storybook/recipes/TableCard/TableCard.tsx

```diff
@@ -75,20 +75,14 @@ export const TableCard = ({
         >
           {title}
         </Heading>
-        <Table
-          caption="Standards coverage"
-          className={styles['table-card__table']}
-          hideCaption={true}
```

Comment:
`caption` and `hideCaption` props removed, and replaced here for `title` attribute